include/exadg/structure/time_integration/time_int_gen_alpha.cpp

```diff
@@ -124,7 +124,24 @@ TimeIntGenAlpha<dim, Number>::do_timestep_solve()
   VectorType const_vector, rhs;
   const_vector.reinit(displacement_n);
   rhs.reinit(displacement_n);
-  this->compute_const_vector(rhs, displacement_n, velocity_n, acceleration_n);
+
+  // add const_vector resulting from acceleration and weak damping terms
+  double total_scaling_factor_mass = this->get_scaling_factor_mass_from_acceleration();
+  this->compute_const_vector_acceleration_remainder(rhs /* placeholder */,
```

Comment:
I do not understand the "placeholder" comment here (and below). The vector is changed inside the function call, so I would not say it is a "placeholder".